x-pack/plugins/security/server/usage_collector/security_usage_collector.ts

```diff
@@ -53,26 +53,49 @@ export function registerSecurityUsageCollector({ usageCollection, config, licens
     schema: {
       auditLoggingEnabled: {
         type: 'boolean',
+        _meta: {
+          description:
+            'Indicates if audit logging is both enabled and supported by the current license.',
+        },
       },
       loginSelectorEnabled: {
         type: 'boolean',
+        _meta: {
+          description: 'Indicates if the login selector UI is enabled.',
+        },
       },
       accessAgreementEnabled: {
         type: 'boolean',
+        _meta: {
+          description:
+            'Indicates if the access agreement UI is both enabled and supported by the current license.',
+        },
       },
       authProviderCount: {
         type: 'long',
+        _meta: {
+          description:
+            'The number of configured auth providers (including disabled auth providers).',
+        },
       },
       enabledAuthProviders: {
         type: 'array',
         items: {
           type: 'keyword',
+          _meta: {
+            description:
+              'The types of enabled auth providers (such as `saml`, `basic`, `pki`, etc).',
```

Comment:
This is _unique_ auth providers, right? In that case perhaps this would be a better description:
```suggestion
              'The unique types of enabled auth providers (such as `saml`, `basic`, `pki`, etc).',
```

Reply:
Yours is technically more accurate, but I'm not sure it does much to improve the overall understanding of the field. I'm ultimately indifferent, so if you think it makes it clearer I'll go ahead and make the change

Reply:
> technically more accurate
 
![image](https://user-images.githubusercontent.com/5295965/114087437-e4195280-9881-11eb-85b8-1bd06789b63d.png)

In all seriousness I think it's fine as-is!